docs/README.md

```diff
@@ -86,6 +86,7 @@ SlevomatCodingStandard (24 sniffs)
 ----------------------------------
 - SlevomatCodingStandard.Arrays.TrailingArrayComma
 - SlevomatCodingStandard.Classes.ClassConstantVisibility
+- SlevomatCodingStandard.Classes.ModernClassNameReference
```

Comment:
Dont edit this manually, run composer docs to generate, so the total number is correct.

Reply:
So much for a quick edit using GH UI :stuck_out_tongue: